### PR TITLE
Create README

### LINKED ISSUE #141: Create Readme

### PR DESCRIPTION
**Pull request by AymanSami98**

### CODE REVIEW

**Review by AhmedQeshta — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,164 @@
...
+- [AWS S3](https://aws.amazon.com/s3/)
+- [Twilio](https://www.twilio.com/)
+- [Husky](https://www.npmjs.com/package/husky)
+- [Github Actions](https://github.com/GSG-G11/Fitness-house/actions)
+- [Nodemailer](https://nodemailer.com)
+
+
+### Database Schema <span id="database-schema"></span>
+
+![](https://i.imgur.com/iCpRHGl.png)
```
Comment (AhmedQeshta) [inline README.md:77]:
```suggestion
![database schema](https://user-images.githubusercontent.com/38624002/170699743-9727eb3c-be21-497d-b236-61b2fe2670be.png)

```

**Review by AhmedQeshta — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AhmedQeshta) [reply README.md:77]: we are update our schema database

**Review by AhmedQeshta — APPROVED** [review]

**Review by LinaYahya — COMMENTED** [review]
Great Job Ayman,
Just put the user journy in the first place before user stories, 
And for the tecnologies maybe you can group it , [front, back, CI/CD, storage]

**Review by rawaneyad — APPROVED** [review]
👍🏼

**Pull request merged**